### ISSUE TITLE
Clusters in database schema

### ISSUE THREAD

**Issue by wtmenten** [UDubiSchool/tedsrate#5]

**Comment by timca1** [comment]
There are 6 Criteria each with a set of Values. We combined the values within each criteria into 2 clusters. Below are 6 sections for the 6 criteria. Then I numbered the 2 clusters in each to make it easier to see the organization.

Criteria: Ease of Use  
Cluster 1) Navigation and Findability  
• Browsing/browsability/searchability
• Mediation
• Orientation
• Simplicity

Cluster 2) Structure  
• Formatting/Presentation
• Order/Consistency
• Accessibility

Criteria: Noise Reduction  
Cluster 1) Identity 
• Item Identification
• Subject description/classification/controlled vocabulary
• Subject Summary
• Precision/(relevant retrieved) over (retrieved)
• Selectivity

Cluster 2) Parsimony  
• Linkage / Referral
• Order
• Novelty

Criteria: Quality  
Cluster 1) Completeness 
• Accuracy
• Comprehensiveness
• Currency

Cluster 2) Trustworthiness  
• Reliability
• Validity
• Authority

Criteria: Adaptability  
Cluster 1) Interaction  
• Contextuality/closeness to problem
• Transaction
• Feedback
• Community/social networking
Cluster 2) Customization  
• Flexibility
• Simplicity
• Trust
• Individualization
• Localization
• Privacy

Criteria: Performance  
Cluster 1) Savings  
• Cost savings
• Time savings

Cluster 2) Confidence
• Security
• Safety

Criteria: Affection 
Cluster 1) Attractiveness  
• Aesthetics
• Satisfaction/rewarding/incenting
Cluster 2) Enjoyment  
• Entertainment
• Engagement
• Stimulation
